wasm/src/main/scala/wasm4s/WasmContext.scala

```diff
@@ -95,11 +96,11 @@ trait ReadOnlyWasmContext {
     vtablesCache.getOrElseUpdate(
       name, {
         val functions =
-          collectMethods(name, includeAbstractMethods = true)
+          collectVTableMethods(name, includeAbstractMethods = true)
             .foldLeft(Array.empty[WasmFunctionInfo]) { case (acc, m) =>
               acc.indexWhere(_.name.simpleName == m.name.simpleName) match {
                 case i if i < 0 => acc :+ m
-                case i          => acc.updated(i, m)
+                case i          => if (m.isAbstract) acc else acc.updated(i, m)
```

Comment:
Ah, so is this the fix?

Reply:
It's the fix for abstract methods sometimes getting priority over concrete methods, yes.